setup.py

```diff
@@ -432,6 +434,7 @@ def run(self):
 install_requires = [
     deps["filelock"],  # filesystem locks, e.g., to prevent parallel downloads
     deps["huggingface-hub"],
+    deps["kernels"],  # download kernels from the Hub
```

Comment:
That's the only place I'd want to wait @LysandreJik's input on! IMO this is the longterm plan for sure, wondering if we are gonna do that slowly (first optional soft dep) or not!